src/onFCP.ts

```diff
@@ -42,9 +42,7 @@ export const onFCP = (onReport: FCPReportCallback, opts?: ReportOpts) => {
   const handleEntries = (entries: FCPMetric['entries']) => {
     (entries as PerformancePaintTiming[]).forEach((entry) => {
       if (entry.name === 'first-contentful-paint') {
-        if (po) {
-          po.disconnect();
-        }
+        po!.disconnect();
```

Comment:
You'd think `po` must exist by the time you get to here (otherwise how was it called!), but clearly that wasn't the case previously so what if there are other bugs? Should we just optionally chain here instead just to be sure?

```suggestion
        po?.disconnect();
```

Reply:
I don't think we need to do that. The fact that `po` wasn't defined here was caught by the tests, so I'm not concerned that there are other bugs lurking that we don't know about.

Note that the reason it was in a conditional before had nothing to do with this bug, it had to do with the fact that, in Safari, we used performance timeline rather than performance observer to get FCP. As soon as I refactored to use performance observer, the tests started failing in Safari, making this bug apparent.

Reply:
> Note that the reason it was in a conditional before had nothing to do with this bug…

Yes, that confused me. Gotcha now.